src/NewCommand.php

```diff
@@ -38,8 +38,23 @@ class NewCommand extends Command
      * The boilerplate namespace of the plugin
      * @var string
      **/
+    
+    const PLUGIN_SLUG = 'plugin-name';
+
     const PLUGIN_NAMESPACE = 'Vendor\PluginName';
 
+    const DEF_PLUGIN_DESCRIPTION = 'Description: A description of this plugin';
```

Comment:
You are correct, and that makes perfect sense. I'll have an updated PR for you soon.